src/interactive-window/editor-integration/codeLensFactory.ts

```diff
@@ -256,8 +263,13 @@ export class CodeLensFactory implements ICodeLensFactory {
                 Commands.DebugStop,
                 Commands.RunCellAndAllBelowPalette
             ];
-            fullCommandList = fullCommandList.filter((item) => !commandsToBeDisabledIfNotTrusted.includes(item));
+        } else if (IsWebExtension) {
```

Comment:
need to inject the boolean IsWebExtension, this is just the symbol, which is always true